src/main/java/ch/njol/skript/bukkitutil/WorldUtils.java

```diff
@@ -0,0 +1,48 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.bukkitutil;
+
+import ch.njol.skript.Skript;
+import org.bukkit.World;
+import org.bukkit.generator.WorldInfo;
+
+/**
+ * Utility class for Bukkit worlds
+ */
+public class WorldUtils {
+
+	private static final boolean HAS_MIN_HEIGHT = Skript.classExists("org.bukkit.generator.WorldInfo") && Skript.methodExists(WorldInfo.class, "getMinHeight");
+
+	/**
+	 * Get the minimum height of a world.
+	 * <p>
+	 * Starting with MC 1.17, world heights are able to be below y=0 (using data packs)
+	 * and starting with MC 1.18 the world height by default will be y=-64.
```

Comment:
```suggestion
	 * Starting with MC 1.17, minimum world heights are able to be below y=0 (using data packs)
	 * and starting with MC 1.18 the minimum world height by default will be y=-64.
```
or something like that, otherwise it sounds like the world will be -64 blocks tall